pureskillgg_csgo_dsdk/scrubber/scrub_pii.py

```diff
@@ -0,0 +1,162 @@
+"""
+Scrub PII from your data
+"""
+# pylint: disable=invalid-name
+# pylint: disable=c-extension-no-member
+
+from typing import Dict, Union
+from numbers import Number
+import rapidjson
+
+REDACTED = "redacted"
+
+SCRUB_CSDS_PII_CHANNEL_INSTRUCTIONS = [
+    {"channel": "player_name"},
+    {"channel": "header"},
+    {"channel": "player_personal"},
+    {"channel": "player_info"},
+    {"channel": "player_status"},
+]
+
+
+def scrub_csds_pii(manifest: Dict, data: Dict):
+    """Remove events in overtime rounds
+
+    Inputs:
+        manifest (dict): CSDS manifest.
+        data (dict): CSDS data dictionary.
+
+    Returns:
+        Nothing. Data is changed in place.
+    """
+    manifest = replace_job_id(manifest)
+
+    for i in range(len(manifest["channels"])):
+        manifest["channels"][i]["redacted"] = False
+
+    replace_if_exists(data, manifest, "header", "sharecode")
+    replace_if_exists(data, manifest, "header", "demo_id")
+
+    replace_if_exists(data, manifest, "player_name", "name_new")
+    replace_if_exists(data, manifest, "player_name", "name_old")
+
+    replace_if_exists(data, manifest, "player_personal", "name")
+    replace_if_exists(data, manifest, "player_personal", "clan_tag")
+
+    fix_steam_ids(data, manifest)
+
+    fix_commends_and_wins(data, manifest)
+
+    replace_if_exists(data, manifest, "player_status", "ping", replacement=0)
+
+    return manifest
+
+
+def data_exists(data, channel, column):
+    """Check for a channel/column existing"""
+    if channel in data:
+        if column in data[channel].columns:
+            return True
+    return False
+
+
+def replace_if_exists(
+    data, manifest, channel, column, /, replacement: Union[str, bool, Number] = REDACTED
+):
+    """Replace a column with a certain value if it exists"""
+    if data_exists(data, channel, column):
+        data[channel][column] = replacement
+        channel_index, column_index = get_manifest_indexes(manifest, channel, column)
+        manifest["channels"][channel_index]["columns"][column_index][
+            "origin"
+        ] += "-redacted"
```

Comment:
Should this use? Anywhere else?

```suggestion
        ] += f"-{REDACTED}"
```

Reply:
I don't think it should. I think this is more like a key where `REDACTED` is the value assigned to the overwritten data.

Reply:
Though I'm not super set on this, if you want I can change it to use f string